app/controllers/recipefinders_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RecipefindersController < ApplicationController
+  def index
+    @page = 'homepage' # Display variable
+  end
+
+  def show
+    @page = 'nonhomepage'
```

Comment:
What about if the show page is missing some of those parameters?  Like the from parameter is missing?